app/nametag/nametag.tsx

```diff
@@ -3,76 +3,8 @@ import RefreshAPIs from './RefreshAPIs';
 import Switch from '@mui/material/Switch';
 import { alpha, styled } from '@mui/material/styles';
 import { nametags, hands } from '../state';
+import drawNametag from "@/lib/drawNametag";
 
-function drawNametag(): ImageData {
```

Comment:
It may make it easier to land this change without breaking anything to keep a @deprecated stub method that simply calls out to the new one.

Reply:
@blickly Solved. I have added the old code back to keep a deprecated method.

Reply:
It's important to avoid duplicated code, so let's not keep around two copies of code that are so similar. (See https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

In this context, a stub refers to https://en.wikipedia.org/wiki/Glue_code that only does the minimum set-up necessary to call the real implementation.